documents/xaml-suggested-actions/xaml-designer-suggested-actions-extensibility.md

```diff
@@ -132,5 +132,5 @@ It could be used as a hyperlink to documentation. There are two ways to enabled
    
 
 >- If both `DocumentationAttribute`  and `DocumentationProvider` exist for the same type, `DocumentationAttribute`  will be used.
->- `DocumentationAttribute`  will not be searched in parent classes
->- If `DocumentationProvider` has not been added, help URL will be get from first parent class that has this provider, otherwise Type Name label will be static
\ No newline at end of file
+>- `DocumentationAttribute`  will not be searched in parent classes.
+>- If `DocumentationProvider` has not been added, we will use help URL from the first parent class that has this provider, otherwise Type Name label will not be a hyperlink.
```

Comment:
Now just add a "the" in front of "help URL" and "Type name label" and we're good to go.